packages/flutter_driver/lib/src/frame_sync.dart

```diff
@@ -0,0 +1,37 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'message.dart';
+
+/// Enables or disables the FrameSync mechanism.
+class SetFrameSync extends Command {
+  @override
+  final String kind = 'set_frame_sync';
+
+  /// Whether frameSync should be enabled or disabled.
+  final bool enabled;
+
+  SetFrameSync(this.enabled) : super();
+
+  /// Deserializes this command from JSON generated by [serialize].
+  SetFrameSync.deserialize(Map<String, String> json)
+      : this.enabled = json['enabled'].toLowerCase() == 'true',
```

Comment:
Same as above. The data transferred between driver.dart and extension.dart via the vm_service is actually forced to be of type Map<String, String>.